Bebb-Protocol-and-Apps/PWS
# Rework intro page

Mostly UI changes and a few improvements to UX.

To test:
Look at Intro page on UI and ensure everything works as expected.

Look at About page on UI and ensure everything works as expected.

Look at Create page on UI and ensure everything works as expected.
In particular, open the 3 scene previews and ensure that multiple keys will now trigger the fullscreen mode exit (incl. e.g. Alt, Ctrl, Space, Backspace and as previously Esc).

Specifically, take a look at the Topnav and the Footer on the UI and ensure everything looks in place and works as expected.

Make sure that the Login functionality from the Topnav works as expected. Also ensure that you stayed logged in across the different pages (and that they display accordingly).

In a particular Space, ensure the Inspector now opens in the same position in the space (i.e. the view doesn't change to a default position).
